src/Payments.sol

```diff
@@ -762,7 +761,7 @@ contract Payments is
         nonReentrant
         validateRailActive(railId)
         validateRailTerminated(railId)
-        onlyRailClient(railId)
+        onlyRailOperator(railId)
```

Comment:
The point of this was to allow the client to bypass a broken operator/arbitrator (which will generally be the same entity). If the operator needs the ability to do this, I'd just let them change the arbiter.